WooCommerce/Classes/POS/Presentation/TotalsView.swift

```diff
@@ -98,6 +107,9 @@ private extension TotalsView {
                     .redacted(reason: redacted ? [.placeholder] : [])
             }
             .foregroundColor(Color.primaryText)
+            .transition(.opacity)
+            .matchedGeometryEffect(id: "totalFieldView", in: totalsFieldAnimation)
+            .animation(.default, value: totalsViewModel.isShimmering)
```

Comment:
See above

```suggestion
            .animation(.default, value: shimmeringActive)
```

Reply:
I moved transition and animation calls up the chain https://github.com/woocommerce/woocommerce-ios/pull/13437/commits/d0589ac41966564b4f2ee028717e53afa0e1543e